core/lib/sentry.js

```diff
@@ -65,12 +61,26 @@ async function init(opts) {
       dsn: SENTRY_URL,
     });
 
+    /** @type {LH.Config.Settings | LH.CliFlags} */
+    let settings = opts.flags;
+    try {
+      const {resolvedConfig} = await initializeConfig('navigation', opts.config, opts.flags);
```

Comment:
Settings could change for different modes (e.g. throttling method could override to `devtools` in timespan mode.

That being said, we are only using this on CLI for navigation runs so this makes sense.